test/tests/functional/pbs_preemption.py

```diff
@@ -194,3 +194,60 @@ def test_preempt_sort_when_unset(self):
         self.server.expect(JOB, {ATTR_state: 'S'}, id=jid1)
         self.server.expect(JOB, {ATTR_state: 'R'}, id=jid2)
         self.server.expect(JOB, {ATTR_state: 'R'}, id=jid3)
+
+    def test_preempt_retry(self):
+        """
+        Test to make sure that preemption is retried if it fails.
+        """
+        a = {'resources_available.ncpus': 2}
+        self.server.manager(MGR_CMD_SET, NODE, a, id=self.mom.shortname)
+
+        abort_script = """#!/bin/bash
+exit 3
+"""
+        self.abort_file = self.du.create_temp_file(body=abort_script)
+        self.du.chmod(path=self.abort_file, mode=0755)
+        self.du.chown(path=self.abort_file, uid=0, gid=0, runas=ROOT_USER)
+        c = {'$action': 'checkpoint_abort 30 !' + self.abort_file + ' %sid'}
```

Comment:
You don't actually need the %sid since you aren't using it in the script.  All %sid is doing is passing the session ID as the first argument of the script.